brion/plugin/morphologyHDF5.cpp

```diff
@@ -392,34 +447,28 @@ void MorphologyHDF5::writeSections( const Vector2is& sections,
     H5::FloatType structureDT( H5::PredType::NATIVE_INT );
     structureDT.setOrder( H5T_ORDER_LE );
```

Comment:
You can move these two declarations just before line 456 (and do the same in other functions).